app/components/ToggleDisplay.js

```diff
@@ -80,28 +79,17 @@ export default class ToggleDisplay extends Component {
 
   runHelpers = () => {
     if (this.state.helpers) {
```

Comment:
Sorry I was a bit unclear about the description of my understanding of helpers. They are pointers (via CSS selectors) to html elements yeah? And then as I said they are pointers to elements that should also be hidden when we hide the main html elements targeted.